src/init/update.sh

```diff
@@ -222,6 +222,17 @@ getPreinstalledDirByType()
     return 1;
 }
 
+isWazuhInstalled()
+{
+    if [ -f "${1}/bin/wazuh-control" ]; then
+        return 0;
+    elif [ -f "${1}/bin/ossec-control" ]; then
+        return 0;
+    else
+        return 1;
+    fi
+}
+
 ##########
 # Checks if Wazuh is installed by trying with each installation type. If it finds
```

Comment:
Probably we should update this comment. Because after analyzing the flow, I see that now the return value not only depends on whether the directory is already created or not. Now we have to possibility having the directory, but return `1` in case that `wazuh-control` or `ossec-control` is not found.
We should also make an in-depth analysis to be 100% sure that returning `1` with the `PREINSTALLEDDIR` variable set, does not cause any issue in the installation flow.

Reply:
Fixed. The header comment is now updated to reflect this behavior.
The installation was tested when the PREINSTALLEDDIR is found but the control binary isn't found and the behavior is correct.